codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -817,6 +822,15 @@ private void notifyGoingAway() {
             }
         }
 
+        private void closeStreamsGreaterThanLastKnownStream() {
+            Http2Stream[] streams = new Http2Stream[numActiveStreams()];
+            for (Http2Stream stream : activeStreams().toArray(streams)) {
+                if (stream.id() > lastKnownStream()) {
```

Comment:
@nmittler I think I am missing something here. I want to kill/ignore any stream with a higher id than the last stream id. Shouldn't matter which endpoint created it?

@Scottmitch I think actually we should only ever need to check for for the `lastKnownStream` field. That was an error on my side, I just assumed that `lastKnownStream()` returns `lastKnownStream`.